common/cloudant_base.go

```diff
@@ -195,3 +200,13 @@ func GetAuthenticatorFromEnvironment(credentialKey string) (core.Authenticator,
 
 	return core.GetAuthenticatorFromEnvironment(credentialKey)
 }
+
+// buildUserAgent builds the user agent string.
+func buildUserAgent() string {
+	return fmt.Sprintf("cloudant-go-sdk/%s(%s)", Version, getSystemInfo())
```

Comment:
Chop Version to 8 characters.

Reply:
This is our sdk version, it's semver, so
1. it is guaranteed to be less of equal than 8 chars unless we'll get in two digits on major and three digits on minor or patch - don't see us in danger of this anytime soon
1. If we _do_ get into that position chopping part of version will make version invalid as it'll then indicate incorrect version (e.g. 10.10.100 will become 10.10.10) - I don't think we should prioritize shortness of analytics to their correctness.